docker/Dockerfile

```diff
@@ -324,7 +325,6 @@ RUN set -e; \
         [ "$attempt" -gt 0 ]; \
         sudo PIP_INDEX_URL="$PIP_INDEX_URL" python3 -m pip install -U $PIP_MAYBE_BREAK_SYSTEM_PACKAGES \
             pip \
-            wheel \
```

Comment:
It refuses to remove the preinstalled version.

```
#11 7.443   Attempting uninstall: wheel
#11 7.447     Found existing installation: wheel 0.42.0
#11 7.448 ERROR: Cannot uninstall wheel 0.42.0, RECORD file not found. Hint: The package was installed by debian.
```

Reply:
pip wants to install wheel 0.43.0

Reply:
You may need to find some overwriting flags

Reply:
Similar issue here:

- https://github.com/beeware/briefcase/issues/1738